### PR TITLE
Fix: so active state are showing correctly

### PR DESCRIPTION
**Pull request by davidsoderberg**

### CODE REVIEW

File: apps/web/src/pages/templates/workflow/StepActiveSwitch.tsx [hunk 1]
```diff
@@ -2,15 +2,15 @@ import styled from '@emotion/styled';
...
-      render={({ field }) => <StyledSwitch {...field} label={'Step is Active'} />}
-    />
-  );
-};
+export const StepActiveSwitch = ({ control, index }) => (
+  <Controller
+    control={control}
+    name={`steps.${index}.active`}
+    render={({ field: { value, ...field } }) => {
+      return <StyledSwitch {...field} checked={value} label={'Step is Active'} />;
```
Comment (ainouzgali) [inline apps/web/src/pages/templates/workflow/StepActiveSwitch.tsx:10]: Should we add the label for the disabled state also?  'Step is Disabled'

File: apps/web/src/pages/templates/workflow/WorkflowEditorPage.tsx [hunk 2]
```diff
@@ -10,9 +10,7 @@ import { useTemplateController } from '../../../components/templates/use-templat
 import { StepActiveSwitch } from './StepActiveSwitch';
 
 const capitalize = (text: string) => {
```
Comment (ainouzgali) [inline apps/web/src/pages/templates/workflow/WorkflowEditorPage.tsx:12]: It's fine like this, not a big deal at all.  But I'm wondering why aren't we leaving it uncapitalised like we always did. I think it looks better and it fits the other identifiers (in the case of the channels) 
(email, SMS, in_app vs Email, Sms, In_app)

**Review by ainouzgali — APPROVED** [review]
Great job David💃🏼

File: apps/web/src/pages/templates/workflow/WorkflowEditorPage.tsx (hunk 2 above)
Comment (davidsoderberg) [reply apps/web/src/pages/templates/workflow/WorkflowEditorPage.tsx:12]: because the enum is capitalized and if we do not do it here it will not work...

Comment (davidsoderberg) [reply apps/web/src/pages/templates/workflow/WorkflowEditorPage.tsx:12]: Yeah sure we do not follow our standard for enum if we do that thu...

**Pull request merged**